src/components/application_manager/test/hmi_capabilities_test.cc

```diff
@@ -1318,6 +1318,55 @@ TEST_F(HMICapabilitiesTest,
       hmi_apis::FunctionID::UI_GetCapabilities);
 }
 
+TEST_F(
+    HMICapabilitiesTest,
+    ConvertJsonArrayToSoArray_ConvertPrerecordedSpeech_SuccessConvertFromStringToEnum) {
+  ON_CALL(mock_app_mngr_, IsSOStructValid(_, _)).WillByDefault(Return(true));
+
+  EXPECT_CALL(*(MockMessageHelper::message_helper_mock()),
+              CommonLanguageFromString(_))
+      .WillRepeatedly(Invoke(TestCommonLanguageFromString));
+  EXPECT_CALL(*(MockMessageHelper::message_helper_mock()),
+              CommonLightNameFromString(_))
+      .WillRepeatedly(Invoke(TestCommonLightNameFromString));
```

Comment:
@ydementieiev could `SetUpLanguageAndLightCapabilitiesExpectation` be used instead after https://github.com/LuxoftSDL/sdl_core/pull/17 is merged?

Reply:
@mvorobio yes, after #17 finishes the review I'll update this test